src/adapter/db/wishlist.gateway.ts

```diff
@@ -0,0 +1,68 @@
+import { EntityManager } from '@mikro-orm/core';
+import { EntityRepository } from '@mikro-orm/mysql';
+import { InjectRepository } from '@mikro-orm/nestjs';
+import { Inject, Injectable } from '@nestjs/common';
+import { WishlistNotFoundException } from 'src/application/common/error/exception/wishlist.exception';
+import { AddWishlistCommand } from 'src/application/port/in/wishlist/WishlistUseCase';
+import { WishlistDbCommandPort } from 'src/application/port/out/WishlistDbCommandPort';
+import { WishlistDbQueryPort } from 'src/application/port/out/WishlistDbQueryPort';
+import { Wishlist } from 'src/domain/wishlist';
+
+import { WishlistDbEntity } from './wishlist.entity';
+import { mapToWishlist } from './wishlist.mapper';
+
+@Injectable()
+export class WishlistGateway implements WishlistDbCommandPort, WishlistDbQueryPort {
+  constructor(
+    @InjectRepository(WishlistDbEntity)
+    private readonly wishlistRepository: EntityRepository<WishlistDbEntity>,
+    @Inject(EntityManager)
+    private readonly em: EntityManager,
+  ) {}
+
+  async getWishlistById(wishlistId: number): Promise<Wishlist> {
+    const wishlist = await this.wishlistRepository.findOne({ id: wishlistId });
```

Comment:
wishlist gateway에서 id로 가져오는 경우는 간략하게 작성해줘도 괜찮을 것 같아요 ~ 
```suggestion
  async getWishlistById(id: number): Promise<Wishlist> {
    const wishlist = await this.wishlistRepository.findOne({ id });
```

Reply:
수정했습니다!